tools/font-loader-lib/index.js

```diff
@@ -0,0 +1,34 @@
+const fetchFonts = (window, document) => {
```

Comment:
I don't think it's necessary to pass `window` and `document` to `fetchFonts`, I was doing this when using an inline IIFE as it's a convention that delivers a slight performance improvement.

Reply:
If I don't pass them through, they linter fails with `'document' is not defined ` and `'window' is not defined`...